src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java

```diff
@@ -306,27 +307,29 @@ protected void activate(final BundleContext bundleContext, final ResourceResolve
         }
 
         // optimize alias path allow list
-        this.aliasPathAllowList.clear();
         String[] aliasPathPrefix = config.resource_resolver_allowed_alias_locations();
         if ( aliasPathPrefix != null ) {
+            final Set<String> prefixSet = new TreeSet<>();
             for(final String prefix : aliasPathPrefix) {
                 String value = prefix.trim();
-                    if (!value.isEmpty()&&value.startsWith("/")) { // absolute path should be given
-                        if (value.endsWith("/")) {
-                            this.aliasPathAllowList.add(value);
-                        } else {
-                            this.aliasPathAllowList.add(value + "/");
-                        }
+                if (!value.isEmpty()) {
```

Comment:
What happened to the check for absolute path (starts with "/")? Is it not needed anymore? Why?

If you do re-introduce the check, I would log a warning if it is not an absolute path (but keep skipping empty values silently).

Reply:
Check is required, Will update PR along with warning message